src/js/components/DateInput/__tests__/__snapshots__/DateInput-test.tsx.snap

```diff
@@ -8258,37 +8258,57 @@ exports[`DateInput handle focus in FormField 2`] = `
   border: none;
 }
 
-.c5::-webkit-input-placeholder {
+.c6::-webkit-input-placeholder {
   color: #AAAAAA;
 }
 
-.c5::-moz-placeholder {
+.c6::-moz-placeholder {
   color: #AAAAAA;
 }
 
-.c5:-ms-input-placeholder {
+.c6:-ms-input-placeholder {
   color: #AAAAAA;
 }
 
-.c5 ::-webkit-search-decoration {
+.c6 ::-webkit-search-decoration {
   -webkit-appearance: none;
 }
 
-.c5::-moz-focus-inner {
+.c6::-moz-focus-inner {
   border: none;
   outline: none;
 }
 
-.c5:-moz-placeholder,
-.c5::-moz-placeholder {
+.c6:-moz-placeholder,
+.c6::-moz-placeholder {
   opacity: 1;
 }
 
-.c4 {
+.c5 {
   position: relative;
   width: 100%;
 }
 
+.c3 {
+  outline: none;
+  border-color: #6FFFB0;
+}
+
+.c3 >circle,
+.c3 >ellipse,
+.c3 >line,
+.c3 >path,
+.c3 >polygon,
+.c3 >polyline,
+.c3 >rect {
+  outline: none;
+  box-shadow: 0 0 2px 2px #6FFFB0;
+}
+
+.c3 ::-moz-focus-inner {
+  border: 0;
+}
+
```

Comment:
I'm a little confused on this snapshot change

Reply:
this is no longer relevant